pkgs/build-support/docker/default.nix

```diff
@@ -256,9 +256,13 @@ rec {
       '');
     in
     runCommand name {} ''
-      mkdir -p $out
       cd ${result}
-      cp layer.tar json VERSION $out
+      if [ -e json && -e VERSION ] ; then
```

Comment:
This doesn't seem to work. I'm getting 
```
/build/.attr-0: line 2: [: missing `]'
```

Maybe using `[[` and `]]`is appropriate here?

Reply:
Hmm, not sure how that ever worked. I've patched it, and rebased against current head since it was quite out of date. Here's the test script I used:

```
with import <nixpkgs> { };

with dockerTools;

let
  img = pullImage {
    imageName = "hello-world";
    imageDigest =
      "sha256:1b26826f602946860c279fce658f31050cff2c596583af237d971f4629b57792";
    sha256 = "sha256-OBFb2u3K0UbEs2R7oy35Vxc4JnLcn6QgMZBHk6bWDLQ=";
    finalImageName = "hello-world";
    finalImageTag = "latest";
  };
  tar = exportImage {
    fromImage = img;
    diskSize = 30000;
  };
in tar
```